driver/command/command_nix_test.go

```diff
@@ -0,0 +1,176 @@
+// +build !windows
+
+package command
+
+import (
+	"github.com/deislabs/cnab-go/bundle"
+	"github.com/deislabs/cnab-go/bundle/definition"
+	"github.com/deislabs/cnab-go/driver"
+	"github.com/stretchr/testify/assert"
+
+	"os"
```

Comment:
Nit: this import layout looks weird, the standard library imports should be first